.github/workflows/ci.yml

```diff
@@ -340,12 +343,23 @@ jobs:
 
       - name: Build release
         if: |
+          !startsWith(matrix.os, 'centos') &&
           (matrix.job == 'test' || matrix.job == 'bench') &&
           matrix.profile == 'release' && !matrix.use_sysroot &&
           github.repository == 'denoland/deno' &&
           (github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/tags/'))
         run: cargo build --release --locked --all-targets
 
+      - name: Build release
+        if: |
+          startsWith(matrix.os, 'centos') &&
+          (matrix.job == 'test' || matrix.job == 'bench') &&
+          matrix.profile == 'release' && !matrix.use_sysroot &&
+          github.repository == 'denoland/deno' &&
+          (github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/tags/'))
+        uses: centos:latest
```

Comment:
You can remove this line, it's inferred from the build matrix

Reply:
Thanks, what I'm trying to do is to run the cargo build in centos and I think that github doesn't actually support that in their OS build matrix. I'm not sure...

I'm not totally sure but since this didn't actually work this easily, I think what I need to do is to create an all in one `uses: example/cargo-build-centos:latest` step which is a full docker container which does the cargo build.

LIke this one for example:
https://github.com/roles-ansible/check-ansible-centos-centos7-action

Does that sound like the right strategy?